src/media-segment-request.js

```diff
@@ -279,7 +282,9 @@ const transmuxAndNotify = ({
     audioAppendStart: segment.audioAppendStart,
     gopsToAlignWith: segment.gopsToAlignWith,
     isPartial,
+    remux: isMuxed,
     onData: (result) => {
+      result.type = result.type === 'combined' ? 'video' : result.type;
```

Comment:
The muxer returns `combined` when `remux` is set to true. We treat `combined` or `muxed` as a `video` buffer.